library/alloc/src/collections/mod.rs

```diff
@@ -81,6 +82,19 @@ pub enum TryReserveError {
     },
 }
 
+#[cfg(not(no_global_oom_handling))]
+impl TryReserveError {
+    /// Panic, or worse, according to the global handlers for each case.
+    pub(crate) fn handle(self) -> ! {
```

Comment:
It might be useful to expose this outside the crate, but I didn't get into that for now.